### ISSUE TITLE
Should `aria-hidden-focus` consider iframes and their content?

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4024]
Something like this seems like it should fail `aria-hidden-focus`:

```html
<iframe srcdoc="
  <h2>Heading</h2>
  <button>Click me</button>
" title="frame" aria-hidden="true"></iframe>
```

What's interesting is that in testing this in VoiceOver, I can actually just get at the button and have it announced correct. I would assume that's because `aria-hidden` doesn't impact content in the frame itself. We should do some testing to see how this behaves.
